pkg/api/handlers/compat/images_build.go

```diff
@@ -33,7 +34,13 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if hdr, found := r.Header["Content-Type"]; found && len(hdr) > 0 {
-		if hdr[0] != "application/x-tar" {
+		contentType := hdr[0]
+		switch contentType {
+		case "application/tar":
+			logrus.Warnf("tar file content type is  %s, should use \"application/x-tar\" content type", contentType)
```

Comment:
[`application/x-gtar` is also an alias of `application/x-tar`](http://help.dottoro.com/lapuadlp.php). Do we want to add the same behaviour for that header as well? let it work and write a warning.

Reply:
i wouldnt reject a pr for it.  but otherwise, how about we wait until we are asked for it ?

Reply:
I see no harm in implementing this in another PR but i can also live with not implementing it for now. `application/tar` was important for supporting docker-compose, I am not aware of a tool that uses `application/x-gtar`.